ProxyAgent/Program.cs

```diff
@@ -13,9 +13,8 @@ public static void Main(string[] args)
                 .UseKestrel(options =>
                 {
                     options.AddServerHeader = false;
-                    // The default setting is ProcessorCount/2
-                    options.ThreadCount = System.Environment.ProcessorCount;
                 })
+                .UseLibuv(options => options.ThreadCount = System.Environment.ProcessorCount)
```

Comment:
Any reason for making this change?

Reply:
interesting... if the previous syntax is equivalent to this one, I prefer the previous syntax so we don't expose the implementation detail (the fact that kestrel is based on libuv). 
However, I don't know if they are equivalent, I'd like to know

Reply:
.Net core 1's Kestrel already uses libuv internally. For some reason they have taken out the libuv configuration (threadCount) out from Kestrel options. I assume it could be to avoid tight coupling between Kestrel and libuv. But I couldn't find stong documentation for this reasoning. 

https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel?tabs=aspnetcore2x